tox.ini

```diff
@@ -12,8 +12,28 @@ deps =
     ansible22: ansible~=2.2
 
 commands =
+    unit: pytest
     flake8: flake8
     pylint: python setup.py lint
     yamllint: python setup.py yamllint
-    unit: nosetests
     generate_validation: python setup.py generate_validation
+
+[pytest]
+testpaths =
+    # TODO(rhcarvalho): enable these after https://github.com/openshift/openshift-ansible/pull/3414 merges.
+    #roles/lib_openshift/src/test/unit
+    #roles/lib_utils/src/test/unit
+    roles/openshift_master_facts/test
```

Comment:
Yeah, having to remember to add new directories is problematic, e.g., we missed `lib_openshift` and `lib_utils`.

The purpose of specifying the dirs is making test discovery faster. Maybe we can add some pre-test step that updates the configuration programmatically?!